.rr.yaml

```diff
@@ -254,17 +255,17 @@ temporal:
     # Path to the key file
     #
     # This option is required
-    key: ""
+    key: "/ssl/key.pem"
```

Comment:
empty values not allowed for these now, so I added dummy values in them all. Key must be omitted to disable SSL.